### DISCUSSION TITLE
Welcome to Yolo Tracking Discussions!

### DISCUSSION THREAD

**Discussion by mikel-brostrom** [mikel-brostrom/yolo_tracking#419] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about. :question: 
  * Share what you are using this repo for. :rocket: 
  * Share ideas. :sparkles: 
  * Engage with other community members. :speaking_head: 
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Abduallah-Kadry** [comment]
I am working on accident detection project and i want to use this model to track the cars then try to add CNN block (for example Vgg16) to analyze tracks (trajectories) and classify each frame as an accident or non-accident so 
- is this approach OK ?
- what is the output shape of the Trackers

**Reply by mikel-brostrom** [reply]
Accidents are better detected by trajectory analysis IMO: collision, sudden direction change based on direction from other object when colliding...

**Comment by dulanjana92** [comment]
hi, I'm working on a project which predicts pedestrian crossing intentions  I have used the YOLOv8n pre-trained model and trained with my custom dataset for pedestrians. I want to use my trained model for tracking. How do I do that ? When I give the trained model to track.py it is throwing an error saying "Your model name has to contain either yolox, yolo_nas or yolov8". Is it not possible to use a custom model for tracking ? or any other solution for this error ?

**Comment by mikel-brostrom** [comment]
I think it is obvious from the error message that the model name has to contain either `yolox`, `yolo_nas` or `yolov8`. Sadly not all models contain metadata regarding their architecture. Hence I have no option more than extracting this information from the models' name

**Reply by dulanjana92** [reply]
I understood the error message. But I didn't understand the validation from naming though. I thought I cannot use a custom trained model. Anyway got it working. Thank you.

**Comment by Lienes** [comment]
Hi, I am using **yolov10s** and reidentification model for vehicle tracking. Vehicles are driving in two lanes and 2 direction. I am using **imprassoc** with default parameters except the frame_rate = 4. I have a video segments captured by 4fps.  But the problem is that tracking id may repeat over time. For example, the vehicle ID=3 already is gone and after a while another vehicle with ID=3 appears. Is it possible to configure so that ID never repeats ? Should i do that in boxmot side or in ultralytics ? 

Thanks in advance for any help! :)
